powerauth-restful-server-spring/src/main/java/io/getlime/security/powerauth/app/rest/api/spring/controller/v3/EncryptedDataExchangeController.java

```diff
@@ -176,17 +178,59 @@ public String exchangeSignedAndEncryptedDataString(@EncryptedRequestBody String
                                                                PowerAuthApiAuthentication auth) throws PowerAuthAuthenticationException, PowerAuthEncryptionException {
 
         if (auth == null || auth.getUserId() == null) {
-            logger.debug("Signature validation failed");
+            logger.error("Signature validation failed");
```

Comment:
This should not be error, failed signature is a regular status of a user entering an incorrect PIN code.